README.md

```diff
@@ -10,31 +10,38 @@ for images to work together for a “zooming” effect and works regardless of
 parent elements that have `overflow: hidden` or
 [parents with transform properties](https://codepen.io/rpearce/pen/MEyOmb).
 
-As an added bonus, it will let you zoom _anything_ (see the [`Storybook
-Examples`](https://rpearce.github.io/react-medium-image-zoom/) for more).
+As an added bonus, it will let you zoom _anything_ (see the [`Storybook Examples`](https://rpearce.github.io/react-medium-image-zoom/) for more).
 
 [Blog post announcing `v4`](https://robertwpearce.com/announcing-react-medium-image-zoom-v4.html)
 
 ## Links
-* [Installation](#installation)
-* [Basic Usage](#basic-usage)
-* [API](#api)
-* [Migrating From v3 to v4](#migrating-from-v3-to-v4)
-* [Contributors](#contributors)
-* [Storybook Examples](https://rpearce.github.io/react-medium-image-zoom/)
-* [Changelog](./CHANGELOG.md)
-* [Contributing](./CONTRIBUTING.md)
-* [Code of Conduct](./CODE_OF_CONDUCT.md)
+
+- [react-medium-image-zoom](#react-medium-image-zoom)
+  - [Links](#links)
+  - [Installation](#installation)
+  - [Basic Usage](#basic-usage)
+    - [Uncontrolled component (default)](#uncontrolled-component-default)
+    - [Controlled component (`Controlled`)](#controlled-component-controlled)
+  - [API](#api)
+  - [Both uncontrolled & controlled components](#both-uncontrolled--controlled-components)
+  - [Only the controlled component](#only-the-controlled-component)
+  - [Migrating From v3 to v4](#migrating-from-v3-to-v4)
+  - [Contributors](#contributors)
```

Comment:
Is this from some sort of markdown linter?

Reply:
(ping)

Reply:
@rpearce I believe it’s Prettier. Something wrong with my setup?

Reply:
Did the project's linter do this? I've never seen it do markdown linting... I'd prefer if the project were set up to do this for this to happen; otherwise, keeping it the same in this PR would be preferred.

If you have a way to make the project lint the markdown files, it'd be :+1: if that were in a different PR

Reply:
@rpearce My Visual Studio Code setup uses [Prettier - Code formatter](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode). I had to add `prettier.disableLanguages": ["markdown"]` to my user `settings.json` to disable automatic markdown formatting.

Reply:
> If you have a way to make the project lint the markdown files, it'd be 👍 if that were in a different PR

No sure why this is not already done... Investigating...

Reply:
Oh, I'm not using VSCode might be why... mayhaps `eslint` needs to be told in the `lint` script in the package.json?

Reply:
I have never used `eslint` so puzzled in this regard.

Reply:
I'm pretty sure tslint has been deprecated in favor of eslint, which is the defacto JS linter for the past number of years. The JS & TS linting, plus prettier, are all being piped through eslint in this project. If you're not familiar with it, you should get acquainted: https://www.npmjs.com/package/eslint